core-bundle/src/DependencyInjection/Compiler/TwigPathsPass.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\DependencyInjection\Compiler;
+
+use Contao\CoreBundle\Twig\FailTolerantFilesystemLoader;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * @internal
+ */
+class TwigPathsPass implements CompilerPassInterface
```

Comment:
I think we could improve the class name to say what the class does, e.g. `RewireTwigPathsPass` or `AddTwigPathsToFailTolerantLoaderPass` or something alike.

Reply:
Agreed. I changed it ot `RewireTwigPathsPass` for now.

see 9d946c6a8208e8916a5c42cffd00fab7efb06112

Reply:
Sorry for the force-push confusion - I accidentally had dropped the file when renaming :facepalm:.